securestorage/src/main/resources/application.properties

```diff
@@ -0,0 +1,26 @@
+# DATASOURCE CONFIGURATION
+spring.datasource.platform=postgresql
+spring.datasource.url=jdbc:postgresql://localhost:5432/securestorage
```

Comment:
Do we use different databases for authentication module and docs storage module?